src/Symfony/Bundle/FrameworkBundle/DependencyInjection/Configuration.php

```diff
@@ -144,6 +144,10 @@ public function getConfigTreeBuilder()
                         ->end()
                     ->end()
                 ->end()
+                ->scalarNode('ip_retriever')
+                    ->info('Define your custom service to retrieve IP')
+                    ->defaultNull()
```

Comment:
then [FrameworkExtension.php#L79-L81](https://github.com/xavierleune/symfony/blob/feature/ip-retriever/src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php#L79-L81) must be removed